app/views/uploads/new.html.erb

```diff
@@ -169,8 +169,8 @@ ready = function() {
       enabled.push("cat1_r3", "cat3_r1", "pqrs_mu_individual", "pqrs_mu_group")
       disabled.push("cat1_r31", "cat3_r2");
     } else if (year == "2017") {
-      enabled.push("cat1_r31", "cat3_r2");
-      disabled.push("cat1_r3", "cat3_r1", "pqrs_mu_individual", "pqrs_mu_group")
+      enabled.push("cat1_r31", "cat3_r2", "pqrs_mu_individual", "pqrs_mu_group");

```

Comment:
According to jslint all lines should have a semicolon. Looking at the source of the page it looks like the fact that it is in a ruby app doesn't seem to affect the output. Putting semicolons on all lines is probably the safest bet and is the one that jslint doesn't seem to complain about.